backend/src/handlers/payments.ts

```diff
@@ -112,4 +136,119 @@ export default function mountPaymentsEndpoints(router: Router) {
     await orderCollection.updateOne({ pi_payment_id: paymentId }, { $set: { cancelled: true } });
     return res.status(200).json({ message: `Cancelled the payment ${paymentId}` });
   })
-}
+
+  
+  /* 
+  * 
+  * APP TO USER PAYMENT
+  * 
+  */ 
+  
+  // method that searches the database and returns any eligible refunds for the user
+  router.post('/refundable_payment', async (req, res) => {
+    if (!req.session.currentUser) {
+      
+      return res.status(401).json({ error: 'unauthorized', message: "User needs to sign in first" });
+    }
+  
+    /*
+      implement your logic here
```

Comment:
I would add a tag NOTE TO DEVELOPER: to single to the third party developer this comment is for them